bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2653,9 +2716,25 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
    else if ( name == "RooSimultaneous" )
       Utility::AddUsingToClass( pyclass, "plotOn" );
 
-
-// TODO: store these on the pythonizations module, not on gRootModule
-// TODO: externalize this code and use update handlers on the python side
+   else if (name.find("TVec<") != std::string::npos) {
```

Comment:
i'm probably missing something, but is this handling the case of a `TTree` branch of type `TVec`? they do not exist, `TVec`s are written as `std::vector`s

Reply:
The pythonization works that way: When you construct and python object, the class object ist piped through the `Pythonize(class, name)` C++ function, so you can add new methods. But the name is still the name of the C++ class (`...:TVec<dtype>`).